borg/_hashindex.c

```diff
@@ -77,6 +79,7 @@ static int hash_sizes[] = {
 
 #define HASH_MIN_LOAD .25
 #define HASH_MAX_LOAD .75  /* don't go higher than 0.75, otherwise performance severely suffers! */
+#define HASH_MAX_EFF_LOAD .85
```

Comment:
Reasonably close to [this](https://github.com/leventov/Koloboke/blob/0b4898817f41b0820e0d9a2839fb593f112f9edc/lib/impl/src/main/javaTemplates/net/openhft/koloboke/collect/impl/hash/MutableDHash.java#L31) independent result (0.93)